packages/datagateway-dataview/src/views/landing/isis/isisInvestigationLanding.component.tsx

```diff
@@ -415,27 +420,38 @@ const LandingPage = (props: LandingPageProps): React.ReactElement => {
           <Divider orientation="vertical" />
           {/* Short format information */}
           <Grid item xs={6} sm={5} md={4} lg={3} xl={2}>
-            {shortInfo.map(
-              (field, i) =>
-                data?.[0] &&
-                field.content(data[0] as Investigation) && (
-                  <ShortInfoRow key={i}>
-                    <ShortInfoLabel>
-                      {field.icon}
-                      {field.label}:
-                    </ShortInfoLabel>
-                    <ArrowTooltip
-                      title={getTooltipText(
-                        field.content(data[0] as Investigation)
-                      )}
-                    >
-                      <ShortInfoValue>
-                        {field.content(data[0] as Investigation)}
-                      </ShortInfoValue>
-                    </ArrowTooltip>
-                  </ShortInfoRow>
-                )
+            {data && data[0]?.summary && (
+              <SuggestedInvestigationsSection investigation={data[0]} />
             )}
+            <Accordion defaultExpanded disableGutters elevation={0}>
+              <AccordionSummary expandIcon={<ExpandMore />}>
+                {t('investigations.landingPage.extraInfo')}
+              </AccordionSummary>
+              <AccordionDetails sx={{ px: 0.5, py: 0 }}>
+                {shortInfo.map(
+                  (field, i) =>
+                    data?.[0] &&
+                    field.content(data[0] as Investigation) && (
+                      <ShortInfoRow key={i}>
+                        <ShortInfoLabel>
+                          {field.icon}
+                          {field.label}:
+                        </ShortInfoLabel>
+                        <ArrowTooltip
+                          title={getTooltipText(
+                            field.content(data[0] as Investigation)
+                          )}
+                        >
+                          <ShortInfoValue>
+                            {field.content(data[0] as Investigation)}
+                          </ShortInfoValue>
+                        </ArrowTooltip>
+                        z
```

Comment:
Remove this random `z` character

Reply:
I thought the z character was intentional 😅